src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -721,9 +721,23 @@
      * @see FullIdent
      **/
     public static final int METHOD_CALL = GeneratedJavaTokenTypes.METHOD_CALL;
-
     /**
      * Part of Java 8 syntax. Method or constructor call without arguments.
```

Comment:
Updated. I referred to the expression in the [java document by oracle](https://docs.oracle.com/javase/tutorial/java/javaOO/methodreferences.html) this time. English is not my mother language. I may use some words not appropriately sometimes. If there is still something wrong please point it out. :)